core.js

```diff
@@ -20,11 +20,11 @@ async function fromStream(stream) {
 }
 
 async function fromBuffer(input) {
-	if (!(input instanceof Uint8Array || input instanceof ArrayBuffer || Buffer.isBuffer(input))) {
+	if (!(input instanceof Uint8Array || input instanceof ArrayBuffer)) {
 		throw new TypeError(`Expected the \`input\` argument to be of type \`Uint8Array\` or \`Buffer\` or \`ArrayBuffer\`, got \`${typeof input}\``);
 	}
 
-	const buffer = input instanceof Buffer ? input : Buffer.from(input);
+	const buffer = input instanceof Uint8Array ? input : new Uint8Array(input);
```

Comment:
Had to take a extra look at this, but then i realized that you guard yourself with type checking above.

`Buffer.from(input)` dose a little bit more magic than `new Uint8array(input)` like converting strings.
if this is something you also want to support, then a easy way to convert string to uint8 is via `new TextEncoder().encode(str)` 
I just released a package that converts literally anything to Uint8Array with 0 copies if it's of any interest to you... it's called [to-uint8array](https://www.npmjs.com/package/to-uint8array)


----

The `Buffer.isBuffer(input)` would never be called cuz it would be covered by instanceof uint8 check

Reply:
> like converting strings.

We wouldn't want that at least. The API is documented to accept a buffer.

Reply:
So we allow kind of 3 different input's: Buffer, Uint8Array, ArrayBuffer.
I don't think we need to explicitly add a Buffer here,  as Uint8Array is sufficient.

As @sindresorhus indicated, `string` was not something we had in mind as an input.

Reply:
I didn't expect it to work with strings either.

Just wanted to recommend my package cuz it can convert any ArrayBufferViews (including Buffer + DataView) & ArrayBuffer and turn them into uint8arrays 
Quite handy for any typed array